includes/views/advanced-numbers.php

```diff
@@ -33,7 +33,7 @@
 		</p>
 		<p><?php _e( 'Numbers may have been assigned to orders before this.', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
 		<div class="number-search" style="text-align:right;">
-			<input type="search" id="number_search_input" name="number_search_input" value="<?php echo isset( $_REQUEST['s'] ) ? esc_attr( $_REQUEST['s'] ) : ''; ?>">
+			<input type="number" id="number_search_input" name="number_search_input" value="<?php echo isset( $_REQUEST['s'] ) ? esc_attr( $_REQUEST['s'] ) : ''; ?>">
```

Comment:
![imagen](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/assets/153516591/942bffe2-bd01-4fb8-ae87-a8e4132ca8ec)

Is this behaviour intended?

Reply:
Yes, because we are searching for database values that are integers, so I think it helps. What do you think?

Reply:
I can of course add a min value to avoid that. Thanks for catching it.